Index: jamin/controller/Makefile.am
===================================================================
--- jamin.orig/controller/Makefile.am
+++ jamin/controller/Makefile.am
@@ -1,14 +1,12 @@
-plugin_LTLIBRARIES = jamincont_1912.la
-plugindir = @prefix@/lib/ladspa
-jamincont_1912_la_CFLAGS = @OSC_CFLAGS@ -I../src
-jamincont_1912_la_LIBADD = @OSC_LIBS@
-jamincont_1912_la_LDFLAGS = -module -avoid-version -Wc,-nostartfiles
-jamincont_1912_la_SOURCES = jamincont_1912.c
+pkgbin_PROGRAMS = jamincont_1912.so
+pkgbindir = @prefix@/lib/ladspa
+jamincont_1912_so_CFLAGS = @OSC_CFLAGS@ -I../src -fPIC
+jamincont_1912_so_LDADD = @OSC_LIBS@
 
-EXTRA_DIST = jamincont_1912.xml ladspa-swh.dtd xml2c.pl jamincont_1912.c
-#LINK = gcc -nostartfiles -shared -Wl,-Bsymbolic -lc -o $@
+EXTRA_DIST = jamincont_1912.xml ladspa-swh.dtd xml2c.pl
+LINK = gcc -nostartfiles -shared -Wl,-Bsymbolic -lc -o $@
 
-%.c: %.xml
-	./xml2c.pl $*.xml > $*.c
+%.so.c: %.xml
+	./xml2c.pl $*.xml > $*.so.c
 
 .PRECIOUS:      %.c
